src/hotspot/share/prims/forte.cpp

```diff
@@ -563,10 +563,9 @@ extern "C" {
 JNIEXPORT
 void AsyncGetCallTrace(ASGCT_CallTrace *trace, jint depth, void* ucontext) {
 
-  JavaThread* thread;
+  JavaThread* thread = JavaThread::current_or_null();
```

Comment:
As this can be in a signal handling context it needs to be Thread::current_or_null_safe().

Reply:
Also please add a comment before this:

// Can't use thread_from_jni_environment as it may also perform a VM exit check that is unsafe to
// do from this context.